pisek/env/task_config.py

```diff
@@ -612,12 +626,23 @@ def _format_message(err: ErrorDetails) -> str:
     return f"{err['msg']}: '{inp}'"
 
 
-def _convert_errors(e: ValidationError) -> list[str]:
+def _convert_errors(e: ValidationError, config_values: ConfigValuesDict) -> list[str]:
     error_msgs: list[str] = []
     for error in e.errors():
-        error_msgs.append(
-            f"{_get_section_and_key(error['loc'])}\n{tab(_format_message(error))}"
-        )
+        value: Any = config_values
+        for loc in error["loc"]:
+            value = value[loc]
+
+        if not isinstance(value, ConfigValue):
+            location = (
+                value["_section"].location()
+                if "_section" in value
+                else "Global config error:"
+            )
+        else:
+            location = value.location()
+
+        error_msgs.append(f"{location}\n" + tab(_format_message(error)))
```

Comment:
Wouldn't it be clearer if the message looked like this?
```suggestion
        error_msgs.append(f"In {location}:\n" + tab(_format_message(error)))
```

With `location()` changed to output "config..." instead of "Config...".

Reply:
Changed [here](https://github.com/kasiopea-org/pisek/pull/153/files#diff-32f098dcd2b4f7a414a0bba8124f892c4eb4d19a175f845fac8965035032f9b8R42).